lib/Fhp/Model/StatementOfHoldings/Holding.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace Fhp\Model\StatementOfHoldings;
+
+class Holding
+{
+	/**
+     * @var string
+     */
+    protected $isin;
+	
+	/**
+     * @var string
+     */
+    protected $wkn;
+	
+	/**
+     * @var string
+     */
+    protected $name;
+	
+    /**
+     * @var float
+     */
+    protected $price = 0.0;
+
+    /**
+     * @var float
+     */
+    protected $amount = 0.0;
+	
+	/**
+     * @var float
+     */
+    protected $value = 0.0;
+
+    /**
+     * @var \DateTime|null
+     */
+    protected $date;
+
+	/**
```

Comment:
There are still several tab characters. Also our style guide doesn't allow blank lines before `return` statements. Just running `composer run cs-fix` should take care of all of that.

Reply:
composer run cs-fix bricht mit folgender Fehlermeldung ab:

> > php-cs-fixer fix -v --diff
> sh: php-cs-fixer: command not found
> Script php-cs-fixer fix -v --diff handling the cs-fix event returned with error code 127

in welchen Verzeichnis muss es denn ausgeführt werden?

Reply:
Einfach im Repository-Wurzelverzeichnis, oder drunter. Vermutlich musst du vorher `composer install` machen, damit php-cs-fixer überhaupt erst installiert wird.

Reply:
falsche PHP-Version auf dem Server. Nun hat es funktioniert. Habe die Anpassung gleich via git hochgeladen...